custom_components/teslafi/switch.py

```diff
@@ -20,7 +20,7 @@
         name="Steering Wheel Heater",
         entity_registry_enabled_default=False,
         icon="mdi:steering",
-        available=lambda u, v, h: u and _convert_to_bool(v.get("is_climate_on")),
+        available=lambda u, v, h: u and IS_CLIMATE_ON_STATES.get(str(v.data.get("is_climate_on", "None")), None),
```

Comment:
Since this is checked in two places, may as well just add a TeslaFiVehicle property for this, and use that in both places